### ISSUE TITLE
Entity Attributes Attribute List Entity Type Changes

### ISSUE THREAD

**Issue by smross** [SparkDevNetwork/Rock#2235]
### Prerequisites
* [X] Put an X between the brackets on this line if you have done all of the following:
    * Can you reproduce the problem on a fresh install or the [demo site](http://rock.rocksolidchurchdemo.com/)?
    * Did you include your Rock version number and [client culture](https://github.com/SparkDevNetwork/Rock/wiki/Environment-and-Diagnostics-Information) setting?
    * Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ASparkDevNetwork%2FSlack) to see if your bug or enhancement is already reported?

### Description

Filter Option of `Entity Type` does not obey the normal UI expectation for a "blank" value in the `Entity Type` picker (`/page/193`)

### Steps to Reproduce

1. Login to Rock
2. Go to `Admin Tools`
3. Go to `System Settings`
4. Go to `Entity Attributes`
5. Select the drop-down for `Filter Options`
    ![entity attributes _ rock rms - google chrome 2017-06-02 09 11 09](https://cloud.githubusercontent.com/assets/8688424/26729524/7c55667e-4773-11e7-8743-e1ff2ceed525.png)
6. Change from an entity type in the list or `None (Global Attributes)` to the `Blank` setting (no text)
![entity attributes _ rock rms - google chrome 2017-06-02 09 12 14](https://cloud.githubusercontent.com/assets/8688424/26729554/968060d0-4773-11e7-802e-5d96ba644b06.png)
![entity attributes _ rock rms - google chrome 2017-06-02 09 12 44](https://cloud.githubusercontent.com/assets/8688424/26729576/a76cf1ce-4773-11e7-87ed-c9abc5e1d262.png)

7. Select the `Apply Filter` button

**Expected behavior:** 

When selecting the `Blank` entry the expectation is that ALL <b>Entity Attributes</b> would be listed, regardless of Entity Type or Category

**Actual behavior:** 

Only the 'Entity Type' of 'None (Global Attributes)' is applied, apparently as a filter, even though none is selected

### Versions

* **Rock Version:** 6.2, 6.3 Beta, 6.3, 6.4 Beta, 6.4
* **Client Culture Setting:** en-US

**Comment by cabal95** [comment]
Confirmed on our own install. @smross is correct, it appears that a blank filter is treated as `None (Global Attributes)` rather than `show all`.

**Comment by nairdo** [comment]
We've reviewed this change request.  Since the Attributes block now has an entity type selector (if no Entity Type is set via the block's settings), it is not intended to show all attributes system wide without selecting the type.

Also, I recently fixed a bug in this block via https://github.com/SparkDevNetwork/Rock/commit/a437d3e386b6abb5f48c8183db59dcdc9379b85a so it will behave as designed in the 8.5 hotfix.

![image](https://user-images.githubusercontent.com/383807/47597591-f8349800-d944-11e8-975f-f71b16c50dda.png)
